cms/djangoapps/contentstore/api/views/course_validation.py

```diff
@@ -207,7 +208,9 @@ def _certificates_validation(self, course):
         is_activated, certificates = CertificateManager.is_activated(course)
         return dict(
             is_activated=is_activated,
-            has_certificate=len(certificates) > 0,
+            has_certificate=certificates is not None and len(certificates) > 0,
+            # is_enabled=bool(certificates),
+            is_enabled=has_certificates_enabled(course),
```

Comment:
@fysheets / @MichaelRoytman  : Here is how we can add `is_enabled` either by 
```
is_enabled=bool(certificates),
```
OR
```
is_enabled=has_certificates_enabled(course),
```

Let me know which approach makes more sense. I believe based on this change, we need to update the frontend code too and check certificate.is_enabeld for showing/hidding certificates check.